src/EnergyPlus/UnitVentilator.cc

```diff
@@ -2224,7 +2224,13 @@ namespace UnitVentilator {
                 }
             } else {
                 CheckZoneSizing(state, state.dataUnitVentilators->cMO_UnitVentilator, state.dataUnitVentilators->UnitVent(UnitVentNum).Name);
-                OutAirVolFlowDes = state.dataUnitVentilators->UnitVent(UnitVentNum).MaxAirVolFlow;
+                if (state.dataUnitVentilators->UnitVent(UnitVentNum).OAControlType == FixedOAControl) {
```

Comment:
Fun new pattern. Use `auto &unitVent = state.dataUnitVentilators->UnitVent(UnitVentNum);` and then `unitVent` to shorten lines and make code more readable.

Reply:
That would mask this change by changing many lines and then this one change is hard to see (unless of course I commented that this is the only change). Once the correction is agreed upon, yes, I can make that change.

Reply:
100% understand. Just a suggestion for the future.